src/app/api/cron/notifications/monthly-attendance-report/route.tsx

```diff
@@ -0,0 +1,49 @@
+"use server";
+import * as cron from "node-cron";
+import ReactPDF from "@react-pdf/renderer";
+import { NextResponse } from "next/server";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { MonthlyAttendancePdf } from "@/app/(app)/attendance-report/components/MonthlyReportPdf";
+import { getAttendanceReport } from "@/app/(app)/attendance-report/actions/attendance-report";
+
+export async function GET() {
+  try {
+    cron.schedule("0 6 1 * *", async () => {
```

Comment:
Let's define constants for these cron string. They aren't very readable to everyone.

Reply:
Fixed